### ISSUE TITLE
Pad options: Cannot change 2 fingers and 3 fingers associate button

### ISSUE THREAD

**Issue by KevinNadaud** [linuxmint/cinnamon-control-center#159]
Hi !

There some options to change the button associated with 2 or 3 fingers tap (middle button or right button). But they does not work. 
By default it is 2 fingers associated with right button and 3 fingers associated with middle button and it still that what ever your put. 

To swap the buttons I have to type manually the command line:  
`synclient TapButton2=2 TapButton3=3`
By putting this in the startup application it works after reboot but it will be better if the control center can manage that directly.

Regards

**Comment by icarter09** [comment]
@KevinNadaud are you still experiencing this issue?

**Comment by KevinNadaud** [comment]
Yes the issue is still present (middle click with 3 fingers tap and right click with 2 fingers tap) moreover there no more option to set the number of finger for right of middle button, see the screen shot below (sorry it is in french ...).
With the `synclient TapButton2=2 TapButton3=3` command in startup it works to have middle click with 2 fingers tap and right click with 3 fingers tap

![image](https://user-images.githubusercontent.com/18683315/84866424-fec3d600-b079-11ea-8dd4-2553e3a3c895.png)
